helm_deploy/values-dev.yaml

```diff
@@ -55,6 +55,7 @@ generic-service:
    MANAGE_RESTRICTED_PATIENTS_URL: https://manage-restricted-patients-dev.hmpps.service.justice.gov.uk/
    MANAGE_PRISON_VISITS_URL: https://manage-prison-visits-dev.prison.service.justice.gov.uk/
    LEGACY_PRISON_VISITS_URL: https://prison-visits-booking-staff-dev.apps.live.cloud-platform.service.justice.gov.uk/
+   SECURE_SOCIAL_VIDEO_CALLS_URL: https://auth.dev.prisonvideo.com/accounts/login/oauth/redirect/
```

Comment:
Do you want to include URLs for other environments (assuming there are ones!)?

Reply:
I don't have URLS for the other environments yet - but would like to push this into Dev so the PM and others can see how it operates